models/users/admin.py

```diff
@@ -1,9 +1,19 @@
+from db import db
 from models.user import UserModel
+from models.staff_tenant_link import StaffTenantLink
+from nobiru.nobiru_list import NobiruList
 
 
 class Admin(UserModel):
     __mapper_args__ = {"polymorphic_identity": "admin"}
 
+    tenants = db.relationship(
+        "TenantModel",
+        secondary=StaffTenantLink.tablename(),
+        collection_class=NobiruList,
+        viewonly=True,
+    )
+
```

Comment:
The way things are now: This will always return an empty array. An admin cannot be assigned tenants. I had asked Hugh about this, and he said that only join staff (Staff) can be assigned tenants, and that Admin's should not be able to be assigned tenants. So that's what's been implemented.

If this needs to be changed so that both Admin and Staff can be assigned tenants. Then let me know. I did see in the frontend that logic there had implemented that only admins can be assigned staff, which I submitted changes for in one of my PR's. So it seems there may be a disconnect or misunderstanding here somewhere.

Staff has an association with tenants.
https://github.com/codeforpdx/dwellinglybackend/blob/c992a1ed8f881b69ca67a0e5de8b46daa17fb445/models/users/staff.py#L10-L15